cmd/astro-wasm/astro-wasm.go

```diff
@@ -84,15 +84,7 @@ func makeTransformOptions(options js.Value) transform.TransformOptions {
 		sourcemap = "both"
 	}
 
-	site := jsString(options.Get("site"))
-	if site == "" {
-		site = ""
-	}
-
-	projectRoot := jsString(options.Get("projectRoot"))
-	if projectRoot == "" {
-		projectRoot = "."
-	}
+	injectGlobals := jsString(options.Get("injectGlobals"))
```

Comment:
Only comment is around naming: Would something like `astroGlobalArgs` be more clear? `injectGlobals` sounds like it will do something with `globalThis`.

Reply:
I do like `astroGlobalArgs` more, will update to it 👍